rosidl_runtime_rs/src/traits.rs

```diff
@@ -143,3 +144,17 @@ pub trait Message: Clone + Debug + Default + 'static + Send + Sync {
     /// Converts the RMW-native message into an idiomatic message.
     fn from_rmw_message(msg: Self::RmwMsg) -> Self;
 }
+
+pub trait RmwService {
```

Comment:
This trait shouldn't be needed. Instead, we can generate two structs (idiomatic and RMW-native) that both implement `Service`.

Reply:
Messages must have a function that provides a pointer to the underlying type support, which is a different kind of type support from the type support that services provide, hence the need for a separate trait.

Reply:
I know, what I'm getting at is that you have two service traits here, `RmwService` and `Service`, where one would be sufficient. This is what it the generated code should look like imo:
```rust
mod rmw {
  pub struct @(type_name) {...}
  impl Service for @(type_name) { type Request = crate::srv::rmw::@(type_name)Request; ...}
}
pub struct @(type_name) {...}
impl Service for @(type_name) {type Request = crate::srv::@(type_name)Request; ...}
```

Reply:
It should be a different module: `msg::rmw` and `srv::rmw` can coexist, no?

Reply:
I.e. we are already in the `srv` module by virtue of being in `srv.rs.em`.

Reply:
Done in ec36503